Client.xcodeproj/project.pbxproj

```diff
@@ -14744,7 +14817,7 @@
 					"-ObjC",
 					"-lxml2",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = "$(MOZ_BUNDLE_ID)";
+				PRODUCT_BUNDLE_IDENTIFIER = org.mozilla.ios.Fennec;
```

Comment:
Necessary change?

Reply:
No, this is just me being silly. I made a mistake when I was checking something and accidentally committed it. Will fix. :)